_includes/releases/v22.2/v22.2.0-beta.6.md

```diff
@@ -0,0 +1,28 @@
+## v22.2.0-beta.6
+
+Release Date: November 7, 2022
+
+{% include releases/release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v22-2-0-beta-6-sql-language-changes">SQL language changes</h3>
+
+- Fixed a bug that could cause crashes when parsing malformed [changefeed expressions](../v22.2/cdc-transformations.html). [#90844][#90844]
```

Comment:
Seems like this should go in bug fixes section

It is not really a SQL language change but more fixing a crashing bug